services/core-web/src/components/Forms/permits/PreDraftPermitForm.js

```diff
@@ -70,6 +70,9 @@ export const PreDraftPermitForm = (props) => {
     if (isAmendment) {
       props.change("permit_amendment_type_code", permitType);
     }
+    if (props.isCoalOrMineral && !isAmendment) {
```

Comment:
All new coal and mineral permits are automatically exploration?

Reply:
M/C permits are typically Major Mine permits, but when they come in via a NoW they are usually for exploration, which is handled by the regional office. 
We have a lot of users who keep creating M/C permits instead of MX/CX because they don't click that box. so we're defaulting it to exploration and they can uncheck if in the rare case it's a M/C permit via a NoW. 
There is a ticket to enhance this flow, but this will hopefully prevent from creating the wrong kind now.